samples/model-builder/explain_tabular_sample.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+
+from typing import Dict
+
+
+#  [START aiplatform_sdk_explain_tabular_sample]
+def explain_tabular_sample(
```

Comment:
This could be a generic explain sample and doesn't necessarily need to be qualified as tabular.

Reply:
The sheet has me doing explain_tabular_sample, I assume it's used by the tech writers in such a sample.

Reply:
Perhaps if this is explicitly `explain_tabular_sample` we can also print feature attributions?